src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -733,6 +749,7 @@ TEST_F(ApplicationManagerImplTest,
               CreateModuleInfoSO(
                   hmi_apis::FunctionID::BasicCommunication_UpdateAppList, _))
       .WillOnce(Return(sptr));
+  EXPECT_CALL(*mock_rpc_service_, ManageHMICommand(_)).WillOnce(Return(false));
```

Comment:
@ValeriiMalkov return true probably. And check arg of ManageHMICommand

Reply:
@LuxoftAKutsan Fixed in commit cb0fa9a5bd7649dc7322d420554dc10fdaf4adff